app/views/feeds/_form_overrides.html.erb

```diff
@@ -6,6 +6,46 @@
 
     <div class="card-body">
       <div class="row">
+        <div class="col-12 mb-4">
```

Comment:
Indentation was off in this form - but all I did was add the category/subcategory selects, and move the 2 checkboxes down below them (for aesthetic reasons).